buildpack.toml

```diff
@@ -257,8 +257,8 @@ api = "0.7"
     purl = "pkg:generic/bellsoft-jdk@19.0.2?arch=amd64"
     sha256 = "f9a4a94491de43e51e93b5af0b87f53855554ba01f092f1e82d1e32a37af280a"
     stacks = ["io.buildpacks.stacks.bionic", "io.paketo.stacks.tiny", "*"]
-    uri = "https://github.com/bell-sw/Liberica/releases/download/19.0.2+9/bellsoft-jdk19.0.2+9-linux-amd64.tar.gz"
-    version = "19.0.2"
+    uri = "https://github.com/bell-sw/Liberica/releases/download/20%2B37/bellsoft-jdk20+37-linux-amd64.tar.gz"
```

Comment:
Looking good, you just need to also:
* update the CPE & PURL fields to reference 20 instead of 19
* get the new SHA256 value for the 20 download (i think the update dependency workflow will fix the SHA when it runs next but it might be easier to put the right value in now?)
* set the version field to 20.0.0

Reply:
yeah that's right, I did that too quickly.

Should be good now, please let me know and I'll do the other java buildpacks!